adminSiteServer/app.tsx

```diff
@@ -49,7 +50,7 @@ app.use("/api", publicApiRouter.router)
 app.use("/admin/api", apiRouter.router)
 app.use("/admin/test", testPageRouter)
 
-app.use("/admin/build", express.static("itsJustJavascript/webpack"))
+app.use("/admin/assets", express.static("itsJustJavascript/webpack"))
```

Comment:
I also tried just using `/assets` in here, which would've made things easier since we would then have matching paths on both the admin server and the live site.
But that didn't work; does anyone know why? Is it maybe a Cloudflare configuration issue, where we've configured it such that only requests starting with `/admin` are coming through?